build/ax_cxx_compile_stdcxx_17.m4

```diff
@@ -0,0 +1,35 @@
+# =============================================================================
```

Comment:
Are these two "ax_cxx_*" M4 macros copied from Geany?

Reply:
yes, is there a better way to handle this? I'm new to the autotools world, so any recommendations would be appreciated

Reply:
I don't know if there is a better way :(. Asked primarily for curiosity.